account_payment_order/models/account_move_line.py

```diff
@@ -25,6 +25,10 @@ class AccountMoveLine(models.Model):
         string="Payment lines",
     )
 
+    @api.onchange('partner_id')
+    def empty_partner_bank(self):
```

Comment:
Follow guidelines for names and add docstring:

```suggestion
    def _onchange_partner_id(self):
        """Empty partner bank for avoiding inconsistencies."""
```

Reply:
Done! Thanks!